guides/common/assembly_renaming-server-or-smart-proxy.adoc

```diff
@@ -1,4 +1,13 @@
-include::modules/con_renaming-server-or-smart-proxy.adoc[]
+[id="Renaming_Server_or_Smart_Proxy_{context}"]
+= Renaming {ProjectServer} or {SmartProxyServerTitle}
```

Comment:
```suggestion
= Renaming {ProjectServer} or {SmartProxyServer}
```

Reply:
Please do not add any content to assemblies. > Please move+rename your file to "modules/con_...".

Reply:
Also, title attributes were created exactly to deal with capitalisation, so I would advise to keep `SmartProxyServerTitle`. Furthermore, there is `ProjectServerTitle` attribute as well.